backend/src/services/communityMemberService.ts

```diff
@@ -117,6 +118,12 @@ export default class CommunityMemberService {
           fillRelations,
         )
 
+        sendNewMemberNodeSQSMessage(this.options.currentTenant.id, record.id)
+          .then(() => console.log(`New member automation triggered - ${record.id}!`))
+          .catch((err) =>
+            console.log(`Error triggering new member automation - ${record.id}!`, err),
+          )
+
```

Comment:
Yes but the processing will just fail because automation won't be in the database and nothing will happen besides an error logged in the nodejs-microservice serverless function output. 

I guess it would be good to emit after the transaction is committed just in case the queue message get's processed so fast that the activity tries to be loaded before the transaction is committed. Will do that :)